integration/client_get_services_test.go

```diff
@@ -0,0 +1,64 @@
+package integration
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"testing"
+)
+
+var namespace = flag.String("namespace", "myproject", "Openshift namespace (most often Project) to run our integration tests in")
+var executable = flag.String("executable", "mobile", "Executable under test")
+var update = flag.Bool("update", false, "update golden files")
+
+const testPath = "getServicesTestData/"
+
+func TestGetServices(t *testing.T) {
+	tests := []struct {
+		name    string
+		args    []string
+		fixture string
+	}{
+		{"no arguments", []string{"get", "services"}, "no-args.golden"},
+		{"json output", []string{"get", "services", "-o=json"}, "json-output.golden"},
+		{"table output", []string{"get", "services", "-o=table"}, "table-output.golden"},
+	}
+
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			dir, err := os.Getwd()
+			if err != nil {
+				t.Fatal(err)
+			}
+			cmd := exec.Command(path.Join(dir, *executable), test.args...)
```

Comment:
We probably don't have to construct the path to executable on every iteration.

Second, I would probably want to be able to specify a full path to executable in the -executable flag, as it would allow for further integration testing.